README.md

```diff
@@ -1,89 +1,494 @@
+has multiple benefits which compound over time.
++ [x] **Visual** communication including graphics and ***video***.
+
+### 4. Collaboration
+
+<div align="center">
+    <a href="https://unsplash.com/photos/IgUR1iX0mqM"
+      alt="Pair Programming - Photo credit NESA">
+        <img src="https://user-images.githubusercontent.com/194400/71603902-fc66d180-2b56-11ea-8bac-8486990e9ef3.png" width="800">
+    </a>
+</div>
+
+How to work with other people in a team both in-person and remote.
+This is an incredibly valuable transferrable life skill
+that everyone should master.
+See: https://github.com/dwyl/remote-working
+
+
+### 5. UX/Design Thinking
+
+
+
+You don't have to _label_ yourself as a "designer" or go to college for 4 years
+in order to _think_ like a designer. You just have to _observe_ the world
+and think: "_is this **user-friendly**?_"
+
+![steve-jobs-design-is-how-it-works](https://user-images.githubusercontent.com/194400/72974694-5f026480-3dc7-11ea-9d82-e89165f48d49.jpg)
+
+There are plenty of examples of both good and bad design around us.
+You have the _opportunity_ to improve the world
+and the experience of your fellow humans through design.
+What can be more _exciting_ than that?
+
+Read:
++ https://en.wikipedia.org/wiki/Design_thinking
++ https://en.wikipedia.org/wiki/User-centered_design
++ https://uxplanet.org/user-centered-design-process-and-benefits-fd9e431eb5a9
++ https://99percentinvisible.org/article/norman-doors-dont-know-whether-push-pull-blame-design/
+
+
+### 6. Code
+
+#### “_Everybody should learn to program a computer, <br />because it teaches you how to think._” ~ Steve jobs
+
+We think _everyone_ should learn how to (read/write/understand) code
+because it teaches you to think systematically
+and break down problems into small chunks.
+
+[![image](https://user-images.githubusercontent.com/194400/71601935-a6416080-2b4d-11ea-9ad9-70bd5ada490e.png "Photo by Kevin Ku")](https://unsplash.com/photos/w7ZyuGYNpRQ)
+
+@dwyl we write code to make useful things for real people. <br />
+<!-- We don't write code for hedge funds and high frequency traders
+to [help rich people get richer](https://youtu.be/_HDFegpX5gI). -->
+We do not expect that _all_ the people
+doing the Apprenticeship to write code for their _career_.
+In fact we expect people to gravitate
+to the area of creativity they most enjoy
+and is most fulfilling/valuable to them.
+In the same way that reading, writing and math
+is required in all schools
+(_an is widely considered to be "basic literacy"_)
+we feel that _knowing_ how to code is _essential_.
+
+<!--
+In the same way that not _all_ people
+who are learning to code
+as part of primary school curriculum
+in schools will write code for a living.
+-->
+
+#### “_Whether you want to uncover the secrets of the universe, or you just want to pursue a career in the 21st century, basic computer programming is an essential skill to learn_.” ~ Stephen Hawking
+
+Along the way you
```

Comment:
I don't think this sentence adds too much and would suggest removing it and ending on the quote above.

Reply:
@iteles then just _remove_ it. I we don't need to have a discussion about it. All I want in this repo is that the _message_ is communicated. If we can do that in a **_single_ paragraph**, I'd be happy to **`delete`** the rest of the text:
### Show up on time _every_ day. <br />Learn _faster_ than you thought was _possible_. <br />_Share_ your _knowledge_ with your peers (and future self) every step of the way. <br /> Use your creativity to solve real human problems. <br />Write `code` like your life depends on it. <br />Stay focussed; time is _scarce_! ⏳ 

But sadly, from _experience_ we often need to err on the side of _repeating_ ourselves. 💭 

## _Conclusion_: Please Remove _Anything_ Superfluous. 🔪